pkg/process/util/signal_nowindows.go

```diff
@@ -13,23 +13,14 @@ import (
 // HandleSignals tells us whether we should exit.
 func HandleSignals(exit chan bool) {
 	sigIn := make(chan os.Signal, 100)
-	signal.Notify(sigIn)
+	signal.Notify(sigIn, syscall.SIGINT, syscall.SIGTERM, syscall.SIGCHLD, syscall.SIGPIPE)
```

Comment:
I think you might've forgotten to push your changes removing `SIGCHLD` and `SIGPIPE` from here

Reply:
@sunhay indeed, just pushed another version, thx :)